Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM node:5.1.1
+FROM ubuntu
```

Comment:
please forgive me as i misunderstood your two comments, no offense intended brother and your feedback is very welcome.

i am happy to either fix the ubuntu version or base this off of node (but in another patch). also, just to clarify, this Dockerfile is the same one that i wrote for master, which is what has been running quran.com in prod for the past ~1 year. i am definitely very happy to apply any improvements you can suggest, as this was the first project i learned Docker for in the first place and would love to have things be in an ideal state.